Assets/Scripts/Environment/Hangar/Battery.cs

```diff
@@ -115,10 +148,12 @@ private void Update()
     {
         if (!PlayerManager.IsPaused 
             && Input.GetKeyDown(PlayerConstants.DropObjectKey)
-            && PlayerIsHolding())
+            && transform.parent.gameObject == PlayerManager.PlayerObject)
         {
             DropBattery();
         }
+
+        Debug.Log("Player facing direction = " +PlayerManager.Instance.PlayerFacingDirection);

```

Comment:
Can put a space between the '+' and the second operand here? It kinda annoys me lol.

Reply:
I just removed the comment altogether.